cannot compile from source

### Description

Overview of your issue here.
Compile from source, ruckig is also from source master branch.
moveit_core compile success, moveit_ros_planning failed.

```
Errors     << moveit_ros_planning:make src/logs/moveit_ros_planning/build.make.010.log                                            
In file included from /src/moveit/moveit/moveit_ros/planning/planning_request_adapter_plugins/src/add_ruckig_traj_smoothing.cpp:37:
/src/moveit/moveit/moveit_core/trajectory_processing/include/moveit/trajectory_processing/ruckig_traj_smoothing.h:41:10: fatal error: ruckig/ruckig.hpp: No such file or directory
   41 | #include <ruckig/ruckig.hpp>
      |          ^~~~~~~~~~~~~~~~~~~
```

### Your environment
* ROS Distro: noetic
* OS Version: e.g. Ubuntu 22.04
* Source master branch

--- a/moveit_core/CMakeLists.txt
+++ b/moveit_core/CMakeLists.txt
@@ -51,8 +51,14 @@ else()
 endif()
 
 find_package(octomap REQUIRED)
+
 find_package(ruckig REQUIRED)
+# work around catkin_package not fetching the interface includes from the target
+# to forward to downstream dependencies. The includes do not need to be added
+# in include_directories below because the target is correctly imported here.
+get_target_property(ruckig_INCLUDE_DIRS ruckig::ruckig INTERFACE_INCLUDE_DIRECTORIES)
 set(ruckig_LIBRARIES "ruckig::ruckig")
+
 find_package(urdfdom REQUIRED)
 find_package(urdfdom_headers REQUIRED)
 

